static/third/bootstrap/css/bootstrap-btn.css

```diff
@@ -361,21 +361,21 @@ fieldset[disabled] .btn-link:focus {
 .btn-large,
 .btn-group-large > .btn {
   padding: 10px 16px;
-  font-size: 18px;
+  font-size: 1.125rem;
   line-height: 1.33;
   border-radius: 0px;
 }
 .btn-small,
 .btn-group-small > .btn {
   padding: 5px 10px;
-  font-size: 12px;
+  font-size: 0.75rem;
   line-height: 1.5;
   border-radius: 0px;
 }
 .btn-mini,
 .btn-group-mini > .btn {
   padding: 1px 5px;
-  font-size: 12px;
+  font-size: 0.75rem;
   line-height: 1.5;
   border-radius: 0px;
 }
```

Comment:
Is it really a good idea to change the bootstrap-btn values?  I would rather exclude `static/third` from our linters to avoid patching third-party code if the changes are nonfunctional, like this one.